sass/_Variables.scss

```diff
@@ -24,13 +24,14 @@ $color-light-grey: #bcc3ca;
 $color-blueish-white-grey: #e6ecf0;
 $color-blueish-white: #f7f8f9;
 $color-light-greyish-blue: #9cb4cb;
-$color-disabled-grey: #ddd;
 // Coveo Partners Colors
 $color-coveo-for-sitecore: #dc291e;
 $color-coveo-for-salesforce: #009ddc;
 $color-green: #4caf50;
-$color-dark-green: hsl(122, 39%, 37%);
+$color-dark-green: #3a833c;
```

Comment:
`#3a833c` is the equivalent color of `hsl(122, 39%, 37%)`

Reply:
For these colors, I'd prefer if we were able to use the "same orange for everything, same green for everything etc..". Meaning, changing the old color value instead of adding a new one.

Unless it creates too much of a drastic changes, or create new accessibility problem.
eg: the old orange is used is a different place with a different background (not white) which makes it too hard to see.

But, if that's not the case, I'd prefer if we adapted the old color.

Reply:
Good point. I just checked and this green is only used for the default radio buttons, meaning that this change would also improve accessibility as a bonus.

I'll change the default green.